p2p/src/authenticated/actors/peer/actor.rs

```diff
@@ -355,12 +364,6 @@ impl<E: Spawner + Clock + ReasonablyRealtime + Rng + CryptoRng> Actor<E> {
                                 debug!(err=?e, "failed to send message to client");
                             }
                         }
-                        Some(wire::message::Payload::Handshake(_)) => {
-                            self.received_messages
```

Comment:
You'll also need to remove some now unnecessary items in `metrics.rs`: https://github.com/commonwarexyz/monorepo/blob/fcafa8376d5c1fdcfe99ee56fb828a11e5a42dec/p2p/src/authenticated/metrics.rs#L24